### ISSUE TITLE
Typo and missing detailed instruction in alldownloads.rst

### ISSUE THREAD

**Issue by agiudiceandrea** [qgis/OLD-QGIS-Website-see-QGIS-Website-for-current-one#1066]
## Description

It seems to me the phrase `With the keyring in place you can add the repository to as /etc/apt/sources.list.d/qgis.sources with following content` contains a typo: `add the repository to as`.

Moreover, I think the instruction about how to create the qgis.sources file and how to add the needed content should also be added.

Page URL: https://qgis.org/en/site/forusers/alldownloads.html

**Comment by keesj** [comment]
I have previously worked on improving this page for Linux user but the page is in a bad state again. The instructions
require  to much reading , thinking and editing of file. 

I believe that the most common use case for Linux users will be a (novice) Ubuntu linux users, wanting to run a stable (lts) release but this is made complicated (even for me who has been running Linux for 20+ years). 

Here are some notes I wrote during a install

The text is saying
> Here you will simply install the latest stable QGIS (3.30.x ‘s-Hertogenbosch) in your Debian or Ubuntu without having to edit config files.
But this no longer true , you need to edit the configuration files to change distro name 

> First install some tools you will need for this instructions:
gnupg and software-properties-common .. these do not need to be installed any more (gpg is required for the packageing system  and softwarproperties-common is no longer required.


On the configuration
> Types: deb deb-src
99.9% of the people do not need a dev-src

> URIs: https://qgis.org/debian
Is kinda correct but not really. In the line above I believe at least the user should be pointed to the ltr by default

>Suites: your-distributions-codename
Here is is clear that you need to edit something . why not provide a working examlple (e.g. ubuntu) or indeed use a script to generate this configuration so people can cut & paste the content in their terminal?

Basically.. the page should simply start with the examples shown later in the page.  and "the quickstart " .can then be moved away as to not confuse people?
